tests/views_integration/Tribe/Events/Views/Full_Site/Archive_EventsTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Tribe\Events\Editor\Full_Site;
+
+use Spatie\Snapshots\MatchesSnapshots;
+use TEC\Events\Editor\Full_Site\Archive_Block_Template;
+use Tribe\Test\Products\WPBrowser\Views\V2\HtmlTestCase;
+
+class Archive_EventsTest extends HtmlTestCase {
+	use MatchesSnapshots;
+
+	public function setUp(): void {
+		parent::setUp();
+		add_filter( 'tribe_editor_should_load_blocks', '__return_true', PHP_INT_MAX );
+		tribe( \Tribe__Events__Editor__Provider::class )->register();
+	}
+
+	public function tearDown(): void {
+		// Remove any added filters and cleanup resources.
+		remove_filter( 'tribe_editor_should_load_blocks', '__return_true', PHP_INT_MAX );
```

Comment:
No need to remove the filter explicitly: filters and actions are reset to their pre-test-case state by the test case.